doc/joss/paper.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 'Ginkgo: A high performance numerical linear algebra library'
+tags:
+  - linear-algebra
+  - hpc
+  - cuda
+  - modern-c++
+  - hip
+  - spmv
+authors:
+  - name: Hartwig Anzt
+    orcid: 0000-0003-2177-952X
+    affiliation: "1, 2" # (Multiple affiliations must be quoted)
+  - name: Terry Cojean
+    orcid: 0000-0002-1560-921X
+    affiliation: 1 
+  - name: Yen-Chen Chen 
+    affiliation: 4 
+  - name: Goran Flegar
+    orcid: 0000-0002-4154-0420
+    affiliation: 3 
+  - name: Fritz G{\"o}bel
+    affiliation: 1 
+  - name: Thomas Gr{\"u}tzmacher 
+    affiliation: 1
+  - name: Pratik Nayak 
+    orcid: 0000-0002-7961-1159
+    affiliation: 1 
+  - name: Tobias Ribizel 
+    affiliation: 1 
+  - name: Yu-Hsiang Tsai 
+    orcid: 0000-0001-5229-3739
+    affiliation: 1 
+affiliations:
+ - name: Karlsruhe Institute of Technology
+   index: 1
+ - name: Innovative Computing Laboratory, University of Tennessee, Knoxville 
+   index: 2
+ - name: University of Jaume I 
+   index: 3
+ - name: The University of Tokyo 
+   index: 4
+date: 6th March, 2020.
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Ginkgo is a production-ready sparse linear algebra library for high performance 
+computing on GPU-centric architectures with a high level of performance portability
+and software sustainability.
```

Comment:
Maybe *focus on* software sustainability? Because in my understanding it is not the library that has sustainability but mainly our development processes.